news/895.bugfix

```diff
@@ -0,0 +1,3 @@
+- Change the name of the profile blocks to volto-block, so that the install
```

Comment:
Typo: it is `volto-blocks` plural.
Could be fixed after merge too.

Reply:
@mauritsvanrees Done!